src/ceph.in

```diff
@@ -1267,8 +1267,12 @@ def main():
     return 0
 
 if __name__ == '__main__':
-    retval = main()
-    # shutdown explicitly; Rados() does not
-    if cluster_handle:
-        run_in_thread(cluster_handle.shutdown)
-    sys.exit(retval)
+    try:
+        retval = main()
+        # shutdown explicitly; Rados() does not
+        if cluster_handle:
```

Comment:
shall we put this block in `finally:`?

Reply:
we could.. i wasn't sure if it would accomplish anything tho

Reply:
right. i recommend this to follow b038e8fbf9103cc42a4cde734b3ee601af6019ea -- just because it's more correct..